### PR TITLE
Extra registration link above the fold in setting-up-account.md

### PR DESCRIPTION
**Pull request by gocentas**
Added an extra link to console/registration above the fold, beautified the existing one, plus some wording changes

### CODE REVIEW

File: docs/getting-started/setting-up-account.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Setting up an account
 
-It couldn't get easier than this. Simply open this website and choose one of the three login options:
+It couldn't get easier than this. Simply [open our website](https://console.cast.ai/signup) and choose one of the login options:
```
Comment (anjmao) [inline docs/getting-started/setting-up-account.md:3]: If this is about signup at the end should be `and choose one of the signup options`
Also could change `Log in with your Google account` -> `Signup with your Google account`
And `Log in with your GitHub account` -> `Signup in with your GitHub account`

File: docs/getting-started/setting-up-account.md [hunk 2]
```diff
@@ -9,4 +9,4 @@ Or you can create an account by entering your email and password:
 
 ![](setting-up-account/login1.png)
 
-That's it! Now log in and start your multi-cloud Kubernetes journey! [Create your free account here!](https://login.cast.ai/login?state=g6Fo2SBZM2tlaDgtVWxSYzBhUlZ4Ukc2S2pmbU4xZmZwbGVGNqN0aWTZIHJPS3pfaldKRHBaekFUVzVvV3htRFUxX2VOX0dSVU5yo2NpZNkgQzN3Sk9hQnZkSUdJY1NwZEhoTGVpTjZzSVZBMWlEaEs&client=C3wJOaBvdIGIcSpdHhLeiN6sIVA1iDhK&protocol=oauth2&redirect_uri=https%3A%2F%2Fconsole.cast.ai%2Fapi%2Fauth&response_type=code&scope=openid%20profile%20email&screen_hint=SIGNUP)
+That's it! [Create your free account here](https://console.cast.ai/signup) and start your multi-cloud Kubernetes journey.
```
Comment (anjmao) [inline docs/getting-started/setting-up-account.md:12]: Not sure about this text here as it points to signup again. As this whole `Setting up an account` guide talks only about signup let's make it consistent and follow only signup flow. login1.png when also should be changed to signup pic.

File: docs/getting-started/setting-up-account.md (hunk 1 above)
Comment (gocentas) [reply docs/getting-started/setting-up-account.md]: Fair points, thanks, changed as suggested.

File: docs/getting-started/setting-up-account.md (hunk 2 above)
Comment (gocentas) [reply docs/getting-started/setting-up-account.md:12]: Agree on the screenshot aspect, but don't fully understand the rest. Should the bottom link point to log in or should we not have any text at the bottom at all?

**Comment by gocentas** [comment]
Closing this as I've implement all changes in 1 PR. No idea if I could've added to this instead, sorry.

**Pull request closed without merging**